src/pages/app/package/[package].vue

```diff
@@ -31,18 +36,59 @@ async function loadAppInfo() {
       .single()
     app.value = dataApp || app.value
 
-    const date_id = new Date().toISOString().slice(0, 7)
-    const { data } = await supabase
-      .from('app_stats')
-      .select()
+    if (cycleStart && cycleEnd) {
+      const { count: statsCount } = await supabase
+        .from('stats')
+        .select('*', { count: 'exact' })
```

Comment:
You need to use `head: true` to make the request lighter and get only the count not the data